network/dime_message.ksy

```diff
@@ -0,0 +1,61 @@
+meta:
+  id: dime_message
+  title: DIME (Direct Internet Message Encapsulation) Message
+  license: CC0-1.0
+  endian: be
```

Comment:
```suggestion
  bit-endian: be
  endian: be
```

You are using bit-sized integers in the spec and there are actually two layouts available (`bit-endian: be` is default, but that's only for backward compatibility), so you should make it explicit which one you want (see https://doc.kaitai.io/user_guide.html#bit-endian).